.github/pull_request_template.md

```diff
@@ -9,6 +9,6 @@ Fixes # (issue)
 ### Quick checks:
 
 - [ ] I have followed the [Code Guidelines](https://github.com/ConduitIO/conduit/blob/main/docs/code_guidelines.md).
```

Comment:
@ahamidi @lovromazgon WDYT about removing this?

Reply:
I lean towards removing it. For our own connectors, it might make sense to have this check, but even if we don't it's not the end of the world. For 3rd party connectors it does not make sense to have this check, they are free to choose their own guidelines. And given that we're creating a public template I think we should not include the check.

Reply:
Yep, let's drop it.